docs/plans/2026-05-05-editor-frontend-100-completion/design.md

```
@@ -0,0 +1,72 @@
+
+## 비범위
+
+- backend runtime/engine 완성
+- game screen 구현
+- 새 entity 추가
+- 전체 editor redesign
+- normalizer 제거 또는 backend validation 강화
+
+## 완료 선언 기준
+
+#375와 #376이 merge되고 감사표가 최신화되면 “에디터 프론트 1차 목표 100%”로 선언한다. 그 이후에는 2차 목표인 backend/runtime/game screen으로 이동한다.
```

markdownlint-cli2 (0.22.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)